<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageEditDialog</class>
 <widget class="QDialog" name="ImageEditDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prepare image for analysis</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="100,0,0,0">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <item row="5" column="2" colspan="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3">
    <widget class="HistogramWidget" name="histogram" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>10</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="3">
    <widget class="QCheckBox" name="checkInvert">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Invert colors</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Adjust levels:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="11">
    <widget class="QGraphicsView" name="imageView"/>
   </item>
   <item row="9" column="2" colspan="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QCheckBox" name="checkDesaturate">
     <property name="text">
      <string>&amp;Desaturate image</string>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Basic operations:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <item>
      <widget class="QSpinBox" name="levelMin">
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="levelMax">
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistogramWidget</class>
   <extends>QWidget</extends>
   <header location="global">include/histogramwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImageEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImageEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkInvert</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ImageEditDialog</receiver>
   <slot>invertBoxChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkDesaturate</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ImageEditDialog</receiver>
   <slot>checkBoxChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>levelMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImageEditDialog</receiver>
   <slot>minLevelChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>levelMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImageEditDialog</receiver>
   <slot>maxLevelChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkBoxChanged(int)</slot>
  <slot>invertBoxChanged(int)</slot>
  <slot>minLevelChanged(int)</slot>
  <slot>maxLevelChanged(int)</slot>
 </slots>
</ui>
